TLM/TLM/Patch/_InfoManager/SetModePatch.cs

```diff
@@ -15,7 +15,10 @@ public static class SetModePatch
         [UsedImplicitly]
         public static void Prefix(InfoMode mode, SubInfoMode subMode)
         {
-            if (RoadSelectionPanels.Root?.RoadWorldInfoPanelExt != null) {
+            if (!RoadSelectionPanels.Root)
```

Comment:
Road selection panel does not exist in map editor

Reply:
Yes, but why you are passing `true/false` in line 39? 
Shouldn't it be `() => RoadSelectionPanels.RoadWorldInfoPanel.Hide` or something else (that's where exception is raised)?

Reply:
Setmodepatch is changes  UI for integration with road selection panels and does nothing in msp editor no point executing code if it's useless